src/auditor/auditor.py

```diff
@@ -260,6 +260,13 @@ def setup(self):
         In case of multiple calls (ie, for retry()), all data are re-instantiated/initialized.
         """
 
+        #: temp_dir used by fixes.metadata_fix() to hold xml of sde metadata
+        temp_dir = Path(arcpy.env.scratchFolder, 'auditor')
+        if not temp_dir.exists():
+            if self.verbose:
+                print(f'Creating temp directory {temp_dir}...')
+            temp_dir.mkdir()
```

Comment:
The `exists_ok` property is also an option to simplify this code.
```suggestion
            temp_dir.mkdir(exist_ok=True)
```

Reply:
I thought about that, but I wanted to keep the verbose print. That brings up a bigger issue about how/why this message is reported—I think a more sane solution would be properly using log levels and a console handler, but this gets it shipped.